forms-flow-theme/scss/_forms.scss

```diff
@@ -150,16 +148,23 @@ select option:hover {
       color: var(--ff-gray-800);
       
       &.loading {
-        color: var(--ff-gray-800);
+        cursor: default;
       }
-
-      &.clear {
-        color: var(--ff-gray-800);
-        right: var(--spacer-100);
+      .search-spinner {
+        @include spinner-base;
+        width: 0.75rem;
+        height: 0.75rem;
+        font-size: x-small;
```

Comment:
is this a variable ? like var(--x-small ) ? @Bonymol-aot

Reply:
No, x-small is equal to 0.625rem, but  var(--font-size-xs) is 0.875rem